cargo-crev/src/main.rs

```diff
@@ -157,20 +157,19 @@ fn print_ids<'a>(
     db: &ProofDB,
 ) -> Result<()> {
     for id in ids {
-        let mut tmp = String::new();
+        let tmp;
         println!(
             "{} {:6} {}",
             id,
             trust_set.get_effective_trust_level(id),
-            db.lookup_verified_url(id)
-                .map(|url| url.url.as_str())
-                .or_else(|| {
-                    db.lookup_unverified_url(id).map(|url| {
-                        tmp = format!("({})", url.url);
-                        tmp.as_str()
-                    })
-                })
-                .unwrap_or("")
+            match db.lookup_url(id) {
+                UrlOfId::None => "",
+                UrlOfId::FromSelfVerified(url) | UrlOfId::FromSelf(url) => &url.url,
+                UrlOfId::FromOthers(url) => {
+                    tmp = format!("({})", url.url);
+                    &tmp
+                }
```

Comment:
It'd be nice to display all 3 cases separately. Maybe insert between Id and the URL `==` (verified) `~=` (self-reported) `??` (unverified)

Reply:
Sure. Give it a try!